docs/manuals/source/IntroductionAndTutorial/InstallingBareosWebui.rst

```diff
@@ -440,6 +440,68 @@ As an alternative to the method above the Bvfs cache can be updated after each j
 
    We do provide a specific JobId in the *JobId* command argument in this example. Only the *JobId* given by the placeholder %i will be computed into the cache.
 
+Upgrade from Bareos 21 or lower to Bareos 22
+--------------------------------------------
+
+|webui| now requires **php-fpm** instead of Apache **mod-php**. Due to this change there are some manual steps to consider. Before upgrading, Apache mod-php needs to be disabled or removed. After upgrading, php-fpm needs to be enabled. For details please check the following remarks according to your operating system of choice.
+
+Debian, Ubuntu, Univention
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before upgrading
+^^^^^^^^^^^^^^^^
+
+- Disable or remove any PHP module in Apache2
+
+After upgrading
+^^^^^^^^^^^^^^^
+
+- Enable PHP-FPM Apache2 configuration (e.g. `a2enconf php7.3-fpm`, `a2enconf php7.4-fpm` or `a2enconf php8.1-fpm`)
+
+- Restart Apache2 and PHP-FPM
+
+.. code-block:: bareosconfig
+
+   systemctl restart php8.1-fpm
+   systemctl restart apache2
```

Comment:
```suggestion
.. code-block:: shell-session

   root@host:~# systemctl restart php8.1-fpm.service apache2.service
```

This isn't a bareosconfig, but a shell command or shell session.
Also I think it is recommended to write the service type for systemd.

Reply:
Applied, see https://github.com/bareos/bareos/pull/1287/commits/feacf495ddc53db0df4976c7ace0da87d841150e .